frontend/packages/dev-console/src/components/import/git/SourceSecretSelector.tsx

```diff
@@ -49,34 +94,13 @@ const SourceSecretSelector: React.FC<SourceSecretSelectorProps> = ({
             actionTitle: 'Create New Secret',
             actionKey: CREATE_SOURCE_SECRET,
           }}
-          selectedKey={selectedKey}
+          selectedKey={selectedKey.value}
+          title={
+            selectedKey.value === 'create-source-secret' ? 'Create New Secret' : selectedKey.value
```

Comment:
Use a constant for `'create-source-secret'`

Reply:
Done https://github.com/openshift/console/commit/462df22ebef1682c8ec550a84bf3d529bc276c87